loris/webapp.py

```diff
@@ -781,7 +798,7 @@ def _make_image(self, image_request, image_info):
 
     sys.path.append(path.join(project_dp)) # to find any local resolvers
 
-    app = create_app(debug=True) # or 'opj'
+    app = create_app(debug=False) # or 'opj'
```

Comment:
this run_simple() code here isn't meant for production use. Not sure we should change the debug default to false in create_app() - it might look like this is an approved way to run loris in production.

@alexwlchan any thoughts?

Reply:
Yeah, I would leave the default as `debug=true`. Clear sign that this is only for local testing.

Reply:
The problem is that it isn't documented, and it is not clear why some flags in the config file are being overwritten by the `get_debug_config` method. This function is my main concern. I do not think it is a problem to keep the debug as true if we honour the data from the config file, and not rewrite it.